### PR TITLE
[ch22027] PD Progress tab

### PR DESCRIPTION
**Pull request by danNordlogic**
[ch22027] PD Progress tab

### CODE REVIEW

File: intervention-progress/styles/fr-warnings-styles.ts [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import {html} from '@polymer/polymer/polymer-element.js';
```
Comment (acory) [inline intervention-progress/styles/fr-warnings-styles.ts:1]: reuse from `common` folder

File: intervention-progress/styles/pmp-icons.ts [hunk 2]
```diff
@@ -0,0 +1,87 @@
+import '@polymer/iron-iconset-svg/iron-iconset-svg.js';
```
Comment (acory) [inline intervention-progress/styles/pmp-icons.ts:1]: place file above for reuse (common)

File: intervention-tabs.ts [hunk 3]
```diff
@@ -108,6 +108,7 @@ export class InterventionTabs extends LitElement {
         </intervention-management>
         <intervention-attachments ?hidden="${!this.isActiveTab(this.activeTab, 'attachments')}">
         </intervention-attachments>
+        <intervention-progress ?hidden="${!this.isActiveTab(this.activeTab, 'progress')}"></intervention-progress>
```
Comment (acory) [inline intervention-tabs.ts:111]: you might want to not added to the dom based on isUnicefUser flag

**Review by acory — APPROVED** [review]

File: common/mixins/endpoints-mixin.ts [hunk 4]
```diff
@@ -0,0 +1,234 @@
...
+
+    @property({type: Object})
+    currentUser!: User;
+
+    public endStateChanged(state: any) {
+      if (!isJsonStrMatch(state.commonData!.PRPCountryData, this.prpCountries)) {
+        this.prpCountries = [...state.commonData!.PRPCountryData];
+      }
+      if (!isJsonStrMatch(state.commonData!.currentUser, this.currentUser)) {
+        this.currentUser = JSON.parse(JSON.stringify(state.commonData!.currentUser));
```
Comment (acory) [inline common/mixins/endpoints-mixin.ts:27]: keep this in mind https://app.clubhouse.io/unicefetools/story/22121/store-user-in-redux-pmp

**Review by lajoskovago — APPROVED** [review]

**Pull request merged**